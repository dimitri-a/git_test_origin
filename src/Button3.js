import React from 'react'

import {Button2} from './Button2'

export const Button3 = () =>  <div>Hi there <Button2>Hi there</Button2></div>
