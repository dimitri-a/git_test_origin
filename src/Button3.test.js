import React from 'react'

import renderer from 'react-test-renderer'
import 'jest-styled-components'

import {Button3} from "./Button3";

test('button3 test', () => {
    const tree = renderer.create(<Button3/> ).toJSON()
    expect(tree).toMatchSnapshot()
    //console.log(tree);
   //expect(tree).toHaveStyleRule('color', 'white')
   // expect(tree).toHaveStyleRule('background', 'red')
})