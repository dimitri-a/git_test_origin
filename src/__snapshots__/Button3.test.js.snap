// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button3 test 1`] = `
.c0 {
  color: red;
}

<div>
  Hi there 
  <button
    className="c0"
  >
    Hi there
  </button>
</div>
`;
